VotingApplication/VotingApplication.Web/Scripts/Directives/ResultsTable/ResultsTable.html

```diff
@@ -0,0 +1,15 @@
+﻿<md-list>
+    <div ng-repeat="result in resultBreakdown">
+        <md-list-item ng-click="showVoters=!showVoters">
+            <md-icon md-font-icon="fa-chevron-right" class="fa" alt="chevron-right"></md-icon>
```

Comment:
Might be worth making the arrow change direction to help indicate which ones are open.